corehq/apps/toggle_ui/views.py

```diff
@@ -31,9 +32,28 @@ def page_url(self):
 
     @property
     def page_context(self):
+        toggles = list(all_toggles())
```

Comment:
if this returns a generator you don't have to convert it a list since you're iterating it over it once